packages/lwc/package.json

```diff
@@ -12,49 +12,68 @@
         "url": "https://github.com/salesforce/lwc/issues"
     },
     "main": "index.js",
-    "module": "dist/engine-dom/esm/es2017/engine-dom.js",
-    "typings": "types.d.ts",
+    "type": "module",
     "license": "MIT",
-    "scripts": {
-        "build": "node scripts/build.js",
-        "dev": "node scripts/build.js --watch"
-    },
     "files": [
-        "dist/",
-        "index.js",
-        "types.d.ts"
+        "*.js"
     ],
+    "dependencies": {
+        "@lwc/aria-reflection": "2.49.1",
+        "@lwc/babel-plugin-component": "2.49.1",
+        "@lwc/compiler": "2.49.1",
+        "@lwc/engine-core": "2.49.1",
+        "@lwc/engine-dom": "2.49.1",
+        "@lwc/engine-server": "2.49.1",
+        "@lwc/errors": "2.49.1",
+        "@lwc/features": "2.49.1",
+        "@lwc/integration-karma": "2.49.1",
+        "@lwc/integration-tests": "2.49.1",
+        "@lwc/module-resolver": "2.49.1",
+        "@lwc/perf-benchmarks": "2.49.1",
+        "@lwc/perf-benchmarks-components": "2.49.1",
+        "@lwc/rollup-plugin": "2.49.1",
+        "@lwc/shared": "2.49.1",
+        "@lwc/style-compiler": "2.49.1",
+        "@lwc/synthetic-shadow": "2.49.1",
+        "@lwc/template-compiler": "2.49.1",
+        "@lwc/wire-service": "2.49.1"
```

Comment:
These will have to be updated to latest package version (with accompanying `yarn.lock` updates) before merge.